src/application/_applicationPage.scss

```diff
@@ -1,7 +1,7 @@
 .ApplicationPage {
   @include pageTopLevelComponent();
 
-  .ApplicationPage__action-buttons {
+  .ApplicationPage__action-button {
```

Comment:
Hmm, this is on the right track as far as appearance goes, but I don't think this is the best way going about it. As is, the `-buttons` class still exists in the code and it contains not one but two buttons, yet this was only given to one of those. Following that, I still think the buttons inside the container can be found by the `-buttons` container class alone (which the `button + button` rule should be doing for their horizontal gap already and probably won't anymore with this change – unless it's a useless rule in the first place). I believe the fix can be implemented primarily here on the CSS side instead.